operator/tests/e2e/e2e_test.go

```diff
@@ -1755,7 +1751,7 @@ func getAnnotationsForNS(ns *v1.Namespace) map[string]string {
 	return annotations
 }
 
-func newUniqueTestId() string {
+func newUniquetestID() string {
```

Comment:
Nit: Probably newUniqueTestID ... I will continue to try to nerd-snipe you into writing a linter that can autofix for this rule :-)

Reply:
🤦🏼 actually the "t" should be capitalized; will fix forward in the next PR